mantidproject/mantid
# Migrate Vesuvio content from scripts repo to mantid repo

The algorithms and scripts relating to Vesuvio have been moved over from script repository to the mantid repository. Tests have been updated as required as well as data for these tests added. This issue also fixes #14994 so please ensure that VesuvioWorkflow is not mentioned in the code.
# To test
- Ensure that the unit and system tests related to these changes all run successfully.
- Ensure that the docs are built and hod some information about the algorithms
  - An additional issue will be opened to improve the documentation for these algorithms including workflow diagrams

Fixes #14993

[Release notes](docs/source/release/v3.7.0/indirect_inelastic.rst)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
